jax/_src/lax/lax.py

```diff
@@ -6913,3 +6914,45 @@ def _canonicalize_axis(axis, num_dims):
   if axis < 0:
     axis = axis + num_dims
   return axis
+
```

Comment:
I'd put this primitive in `jax/_src/lax/linalg.py`.

Reply:
Done.